inc/dashboardcards.class.php

```diff
@@ -0,0 +1,642 @@
+     *
+     * @param array $params default values for
+     * - 'title' of the card
+     * - 'icon' of the card
+     * - 'apply_filters' values from dashboard filters
+     *
+     * @return array
+     */
+    public static function nbCreditsRemaining(array $params = []): array
+    {
+        $default_params = [
+            'label' => "",
+            'icon' => self::getIconWallet(),
+            'apply_filters' => [],
+            'alt' => 'Quantity of credit remaining',
+        ];
+        $params = array_merge($default_params, $params);
+
+        $tab = self::getCredits($params);
+
+        $result = $tab['quantity'] - $tab['sum'];
+
+        $result = $result < 0 ? 0 : $result;
+
+        return [
+            'number' => $result,
+            'url' => '',
+            'label' => 'Quantity of credit remaining',
+            'icon' => $default_params['icon'],
+            'alt' => $default_params['alt']
+        ];
+    }
+
+    /**
+     * @param array $params
+     * @return array
+     */
+    public static function percentCreditsUsed(array $params = []): array
+    {
+        $default_params = [
+            'label' => "",
+            'icon' => self::getIconWallet(),
+            'apply_filters' => [],
+            'alt' => 'Proportion of credit used',
+        ];
+        $params = array_merge($default_params, $params);
+
+        $tab = self::getCredits($params);
+
+        $result = $tab['quantity'] != 0 ? (($tab['sum']) / $tab['quantity']) * 100 : 0;
+
+        return [
+            'number' => round($result, 0, PHP_ROUND_HALF_DOWN) . '%',
+            'url' => '',
+            'label' => 'Proportion of credit used',
+            'icon' => $default_params['icon'],
+            'alt' => $default_params['alt']
+        ];
+    }
+
+    /**
+     * @param array $params
+     * @return array
+     */
+    public static function percentCreditsRemaining(array $params = []): array
+    {
+        $default_params = [
+            'label' => "",
+            'icon' => self::getIconWallet(),
+            'apply_filters' => [],
+            'alt' => 'Proportion of credits remaining',
+        ];
+        $params = array_merge($default_params, $params);
+
+        $tab = self::getCredits($params);
+
+        $result = $tab['quantity'] != 0 ? (($tab['quantity'] - $tab['sum']) / $tab['quantity']) * 100 : 0;
+
+        $result = $result < 0 ? 0 : $result;
+
+
+
+        return [
+            'number' => round($result, 0, PHP_ROUND_HALF_DOWN) . '%',
+            'url' => '',
+            'label' => 'Proportion of credits remaining',
+            'icon' => $default_params['icon'],
+            'alt' => $default_params['alt']
+        ];
+    }
+
+    /**
+     *
+     * @param array $params default values for
+     * - 'title' of the card
+     * - 'icon' of the card
+     * - 'apply_filters' values from dashboard filters
+     *
+     * @return array
+     */
+    public static function getCreditsConsumption(array $params = []): array
```

Comment:
For this widget 

![image](https://github.com/user-attachments/assets/091b0d00-ff25-4157-bd78-09dbeae42e42)

we need to go further in data representation

on ticket 116 I have 

2 credit used for "Assistance" and 3 for "Bug"

![image](https://github.com/user-attachments/assets/1fc92272-8e9f-4466-8da2-6460f7963907)

I should therefore have 2 columns for ticket 116